npc/common/src/main/java/org/screamingsandals/lib/npc/NPCImpl.java

```diff
@@ -60,8 +65,27 @@
     private volatile boolean lookAtPlayer;
     private final List<MetadataItem> metadata;
 
+    
     private SClientboundSetPlayerTeamPacket.CollisionRule collisionRule;
-
+    private static boolean IS_BUNGEECORD_ENABLED;
+    static {
+        File spigotYmlFile = new File("spigot.yml");
+        IS_BUNGEECORD_ENABLED = false;
+        java.util.regex.Pattern pattern = Pattern.compile("bungeecord: ?(y|yes|true|on)",Pattern.CASE_INSENSITIVE);
+
+        if (PluginManager.getPlatformType() == PlatformType.BUKKIT && spigotYmlFile.isFile()) {
+            try (Scanner scanner = new Scanner(spigotYmlFile)) {
+                while (scanner.hasNextLine()) {
+                    String line = scanner.nextLine().toLowerCase();
+
+                    if (pattern.matcher(line).matches())
```

Comment:
`Pattern#matches()` is not going to work, since [it attempts to match the entire region against the pattern](https://docs.oracle.com/javase/7/docs/api/java/util/regex/Matcher.html#matches()) and the line has some indentation in the start, so replace it with `Pattern#find()`

use
```java
if (condition) {
    // code
}
```
instead of
```java
if (condition)
    // code
```

Reply:
since there is no $ symbol for start and end, it just check it, I tested and it works with matches, but I'm changing to find